src/SDWaveFile.cpp

```diff
@@ -39,6 +39,8 @@ struct WaveFileHeader {
   } subChunk2;
 } __attribute__((packed));
 
+  struct WaveFileHeader header;
```

Comment:
It's not a good idea to make the header a global variable, as a sketch could create multiple Wave File objects.

Reply:
Could be declared static, or can set as parameters for the fucntion findData, what do you think?

Reply:
umm, I'm not sure about making it static, what would happen if multiple wave file objects are created?